src/main/scala/is/hail/variant/View.scala

```diff
@@ -0,0 +1,13 @@
+package is.hail.variant
+
+import is.hail.annotations._
+import is.hail.expr._
+import is.hail.utils._
+
+trait View {
```

Comment:
Does the view buy much beyond giving you the overloaded setRegion method? It seems to me that our codebase should never have anything of type `View`, only the subclasses.

Reply:
This is required for `ArrayView` to be able to work with an element view.